modules/ROOT/nav.adoc

```diff
@@ -25,7 +25,7 @@
  ** xref:flow-counting-studio.adoc[Track the Number of Billable Flows in Studio]
  ** xref:test-specification-api-console.adoc[Test your API Specification Using the API Console]
 
-* xref:api-sync.adoc[API Sync]
+* xref:api-sync.adoc[Sync Your Mule Projects with MuleSoft VCS]
```

Comment:
this only affects API projects, not Mule projects. I would change this to "Sync your API projects with MuleSoft VCS"